script.js

```diff
@@ -1,16 +1,30 @@
-const TODOS_SESSION_KEY = "todos";
+const JSON_STORAGE_URL = "http://158.129.206.250:4000/api/v1";
```

Comment:
this service is no longer available. you need to run it locally so please update url to:
```
const JSON_STORAGE_URL = "http://localhost:4000/api/v1";
```